packages/salesforcedx-vscode-core/src/commands/forceSourceDeploySourcePath.ts

```diff
@@ -130,9 +121,15 @@ export class LibraryDeploySourcePathExecutor extends LibraryCommandletExecutor<
         this.sourceClient.tooling.deploy
       );
 
+      const projectNamespace = (await SfdxProjectConfig.getValue(
+        'namespace'
+      )) as string;
       const registryAccess = new RegistryAccess();
       const components = registryAccess.getComponentsFromPath(response.data);
-      const deployPromise = this.sourceClient.tooling.deploy({ components });
+      const deployPromise = this.sourceClient.tooling.deploy({
```

Comment:
can we create this `deployPromise` after getting the `metadataCount`? Or do we need to create this promise before `await`ing it later?

Reply:
We could, I kept it as it was and just added the `namespace` option to the existing call.

Reply:
The point of it was to have the request running as the map was being built, but if it's confusing you can change it.